reference-architecture/vmware-ansible/roles/openshift-default-selector/tasks/main.yaml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Set the default node selector
+  replace:
+     dest: /etc/origin/master/master-config.yaml
+     regexp: 'defaultNodeSelector: ""'
+     replace: 'defaultNodeSelector: "role=app"'
```

Comment:
setting `osm_default_node_selector` to `role=app` can be used here to configure this during the openshift-ansible run.